Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -221,6 +220,14 @@ def build_iterator(self, **kwargs):
                         f' {r.status_code!r} {r.content!r}')
                     raise
                 url_to_response_list.append({url: r})
+        except requests.exceptions.SSLError as exception:
+            err_msg = 'SSL Certificate Verification Failed - try selecting \'Trust any certificate\' checkbox in' \
+                      ' the integration configuration.'
+            raise DemistoException(err_msg, exception)
+        except requests.exceptions.ProxyError as exception:
+            err_msg = 'Proxy Error - if the \'Use system proxy\' checkbox in the integration configuration is' \
+                      ' selected, try clearing the checkbox.'
+            raise DemistoException(err_msg, exception)
         except requests.ConnectionError:
             raise requests.ConnectionError('Failed to establish a new connection. Please make sure your URL is valid.')
```

Comment:
Let's make sure we keep the original message. By raising a new error we lose the original cause for the connection error. It could be various reasons (dns lookup, timeout).

Reply:
same here